### PR TITLE
cleaned up download.scss

### PR DESCRIPTION
**Pull request by pmahnke**
## Done

* tidied up the _download.scss file 

## QA

1. click through /download section and see that it matches live

### CODE REVIEW

File: static/css/section/_download.scss [hunk 1]
```diff
@@ -1,225 +1,20 @@
...
 
-.download .desktop.box-highlight {
-  margin-bottom: 40px;
-}
+// ---------------------------------------------------------------
+// ---------------------------------------------------------------
+//
+//  Project:		Ubuntu Front-End Download section
+//  Author:			Web Team at Canonical Ltd
+//  Last edited by: Peter Mahnke
```
Comment (nottrobin) [inline static/css/section/_download.scss]: Isn't this and the next line redundant? I don't think there's much chance that we'll remember to keep this up-to-date. Also, we have version control.

Comment (anthonydillon) [inline static/css/section/_download.scss]: +1

File: static/css/section/_download.scss [hunk 2]
```diff
@@ -237,812 +32,614 @@ html.opera-mini {
...
-                border-bottom: 1px dotted $warm-grey;
-                padding-bottom: 20px;
-                margin-bottom: 20px;
-            }
-            ul {
-                margin-bottom: 0;
-            }
-        }
-    }
+  // XXX code stuff
```
Comment (anthonydillon) [inline static/css/section/_download.scss]: Can you give this a better description please?

File: static/css/section/_download.scss [hunk 3]
```diff
@@ -237,812 +32,614 @@ html.opera-mini {
...
-      margin-bottom: 0;
+    a:hover {
+      color: rgba(255, 255, 255, .6);
     }
 
-    .row-wiki {
-        border-bottom: 1px dotted $warm-grey;
-        margin-top: 20px;
+    a:hover.link-cta-download {
+      color: rgba(255, 255, 255, 1);
```
Comment (anthonydillon) [inline static/css/section/_download.scss]: This is the posh way of saying `$whtie` :)

**Comment by anthonydillon** [comment]
Some QA points. Like the way yours looks so ignore if expected.

## QA
/download/server
- border bottom on the row hero

/download/ubuntu-flavours
- Extra padding under row hero

/download/desktop/contribute
- The sliders box is box inside box on live
- Pay with Paypal is full width

**Comment by pmahnke** [comment]
updating, but I don't see the variable white?

File: static/css/section/_download.scss (hunk 3 above)
Comment (pmahnke) [inline static/css/section/_download.scss]: does $white work? there are a _lot_ of #fff's

File: static/css/section/_download.scss (hunk 2 above)
Comment (pmahnke) [inline static/css/section/_download.scss]: hah... that was there... I will remove

File: static/css/section/_download.scss (hunk 1 above)
Comment (pmahnke) [inline static/css/section/_download.scss]: @nottrobin and @anthonydillon removing both Last edited and date

**Comment by pmahnke** [comment]
I fixed flavours and contribute by adding .dowload to the main classes to get more specific...

* .download.download-ubuntu-flavours
* .download.download-desktop-contribute

I wish I didn't have to do that... 

And @anthonydillon I don't see the border bottom on the row hero on live either on /download/server - can you send a screenshot?

File: static/css/section/_download.scss (hunk 3 above)
Comment (anthonydillon) [inline static/css/section/_download.scss]: Yes, should be available from Vanilla.

**Comment by anthonydillon** [comment]
download ubuntu server download ubuntu

**Pull request merged**